src/__tests__/railgun-engine.test.ts

```diff
@@ -1140,7 +1141,7 @@ describe('railgun-engine', function test() {
     const newBalance2 = await wallet2.getBalanceERC20(txidVersion, chain, tokenAddress, [
       WalletBalanceBucket.Spendable,
     ]);
-    expect(newBalance2).to.equal(BigInt(11));
+    expect(newBalance2).to.equal(BigInt(10));
```

Comment:
I'm not 100% sure if this is right, but according to the transactions created above, there is one 10n actually transferred and 1n as relayer fee. So I assume the relayer fee is not counted in this balance, but I'm not very sure.